packages/expo-modules-core/ios/AppDelegates/EXAppDelegateWrapper.m

```diff
@@ -27,6 +27,10 @@ - (instancetype)init
   return self;
 }
 
+- (void)buildMenuWithBuilder:(id<UIMenuBuilder>)builder{
```

Comment:
I believe this is not necessary as this class forwards all method invocations to `_expoAppDelegate` anyway 😉

Reply:
Well, it does not forward this call. It might be that it forwards all AppDelegate callbacks - but not this which is UIResponder. Maybe you have a better suggestion?